migration_steps/validation/counts_verification/count_final.py

```diff
@@ -0,0 +1,92 @@
+import os
```

Comment:
This script is also nearly the same as the previous ones. Is there any benefit in using a single script to reduce the amount of code we need to maintain?

Reply:
As above